models/tenant.py

```diff
@@ -16,17 +17,19 @@ class TenantModel(BaseModel):
     # relationships
     staff = relationship("UserModel", secondary="staff_tenant_links")
     leases = db.relationship(
-        "LeaseModel", backref="tenant", lazy=True, cascade="all, delete-orphan"
+        "LeaseModel", backref="tenant", lazy="dynamic", cascade="all, delete-orphan"
     )
 
     def json(self):
+        first_active_lease = self.leases.filter(LeaseModel.active()).first()
+        active_lease_json = first_active_lease.json() if first_active_lease else ""
```

Comment:
I think it might be better to return None instead of an empty string when there isn't an active lease, but it depends on what the frontend wants.